account_invoice_journal_filter/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Account Invioce Journal Filter
+==============================
+
+This module extends the functionality of account invoice to filter by journal.
+
+
+Installation
+============
+
+To install this module, you need to:
+
+  Just install the module
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+	No configuration needed
+
+
+
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Firstname Lastname <jjs@adhoc.com.ar>
+
```

Comment:
give the firstname and the lastname of the contributor except if those are his real firstname and lastname.